stm32cube/stm32f4xx/drivers/include/stm32f4xx_hal_conf.h

```diff
@@ -333,8 +334,8 @@
 #endif /* HAL_ETH_MODULE_ENABLED */
 
 #ifdef HAL_ETH_LEGACY_MODULE_ENABLED
-  #include "Legacy/stm32f4xx_hal_eth_legacy.h"
-#endif /* HAL_ETH_MODULE_ENABLED */
+  #include "stm32f4xx_hal_eth_legacy.h"
+#endif /* HAL_ETH_LEGACY_MODULE_ENABLED */
```

Comment:
Although we don't use this file we should probably follow this change for consistency.